### PR TITLE
Multigeo site enumeration API support

### PR DESCRIPTION
**Pull request by tushar20**

### CODE REVIEW

**Comment by Danielabom** [comment]
Contacted author to get info.

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:1]: The correct format for the file name is: sites-getallsites

Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:1]: The correct file name for this operation would be: sites-getallsites

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+title: "Enumerate sites across geographies"
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:2]:
```suggestion
title: "sites: getallsites"
```

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+description: "List all root sites across geographies in an organization - OneDrive API"
+author: tkanaujia
+ms.author: tkanaujia
+ms.date: 24/03/2023
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# Enumerate multigeo sites
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:12]:
```suggestion
# sites: getAllSites
```

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+List all available [sites][] across geographies in an organization.
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:14]: Please add the link to the resource. 
```

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+[sites]: ../resources/site.md
+
+For more guidance about building applications that use site discovery for scanning purposes please see https://aka.ms/scanguidance.
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:18]:
```suggestion
For more guidance about building applications that use site discovery for scanning purposes please see (https://aka.ms/scanguidance).
```

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# Enumerate multigeo sites
+
+List all available [sites][] across geographies in an organization.
+
+[sites]: ../resources/site.md
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:16]: I believe this link goes in line 14.

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+author: tkanaujia
+ms.author: tkanaujia
+ms.date: 24/03/2023
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# Enumerate multigeo sites
+
```
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:13]: Please add the beta disclaimer.

File: api-reference/v1.0/api/sites-list-all-rootsites-multigeo.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
+---
```
Comment (Danielabom) [inline api-reference/v1.0/api/sites-list-all-rootsites-multigeo.md:1]: Please see the comments I added for the beta version of this file.

File: api-reference/v1.0/resources/site.md [hunk 9]
```diff
@@ -19,6 +19,7 @@ The **site** resource provides metadata and relationships for a SharePoint site.
 |:-------------------------|:-------------|:----------
 | [Get root site][]        | site | Access the root SharePoint site within a tenant.
 | [Get site][]             | site | Access a sharePoint site using the siteId.
+| [List root sites across geographies][] |  collection of site  | List sites across all geographies in an organization
```
Comment (Danielabom) [inline api-reference/v1.0/resources/site.md:22]:
```suggestion
| [List root sites across geographies][] |  collection of site  | List sites across all geographies in an organization.
```

File: changelog/Microsoft.FileServices.json [hunk 10]
```diff
@@ -1,5 +1,41 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "d07006c1-a451-4fcf-b9b6-835546842783",
+          "ApiChange": "Method",
+          "ChangedApiName": "getAllSites",
+          "ChangeType": "Addition",
+          "Description": "Added the [getAllsites](https://learn.microsoft.com/en-us/graph/api/resources/site?view=graph-rest-beta) method to the site resource to fetch all sites across all geographies in an organization.",
```
Comment (Danielabom) [inline changelog/Microsoft.FileServices.json:10]: Please add the link to the resource. You may use example 7 on this wiki: (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @tushar20, thank you so much for working on this PR. Please review my comments.

File: api-reference/beta/api/sites-list-all-rootsites-multigeo.md (hunk 5 above)
Comment (Danielabom) [inline api-reference/beta/api/sites-list-all-rootsites-multigeo.md:18]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance.md).
```

**Comment by tushar20** [comment]
Hi @Danielabom, updated the PR with the suggested changes. Kindly take a look.

File: api-reference/beta/api/sites-getallsites.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more guidance about building applications that use site discovery for scanning purposes please see <https://aka.ms/scanguidance>.
```
Comment (Danielabom) [inline api-reference/beta/api/sites-getallsites.md:18]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance.md).
```

**Review by Danielabom — CHANGES_REQUESTED** [review]
@tushar20, the changes look good. Now that you have updated the file names. Please update the TOC for both versions accordingly. Here's our TOC wiki: (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211766/ToC-and-topic-title)

**Comment by Danielabom** [comment]
Following up on my previous comment: @tushar20, the changes look good. Now that you have updated the file names. Please update the TOC for both versions accordingly. Here's our TOC wiki: (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211766/ToC-and-topic-title)

**Comment by Danielabom** [comment]
@tushar20, please add the TOC as soon as you can. Thanks.

File: api-reference/beta/api/sites-getallsites.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
+---
+title: "sites: getAllSites"
+description: "List all root sites across geographies in an organization - OneDrive API"
+author: tkanaujia
```
Comment (Danielabom) [inline api-reference/beta/api/sites-getallsites.md:4]:
```suggestion
author: tushar20
```

File: api-reference/beta/api/sites-getallsites.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
+---
+title: "sites: getAllSites"
+description: "List all root sites across geographies in an organization - OneDrive API"
+author: tushar20
+ms.author: tkanaujia
+ms.date: 24/03/2023
```
Comment (Danielabom) [inline api-reference/beta/api/sites-getallsites.md:6]:
```suggestion
ms.date: 5/3/2023
```

File: api-reference/v1.0/api/sites-getallsites.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
+---
+title: "sites: getAllSites"
+description: "List all root sites across geographies in an organization - OneDrive API"
+author: tkanaujia
```
Comment (Danielabom) [inline api-reference/v1.0/api/sites-getallsites.md:4]:
```suggestion
author: tushar20
```

File: api-reference/v1.0/api/sites-getallsites.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
+---
+title: "sites: getAllSites"
+description: "List all root sites across geographies in an organization - OneDrive API"
+author: tushar20
+ms.author: tkanaujia
+ms.date: 24/03/2023
```
Comment (Danielabom) [inline api-reference/v1.0/api/sites-getallsites.md:6]:
```suggestion
ms.date: 5/3/2023
```

File: api-reference/beta/toc.yml [hunk 16]
```diff
@@ -16315,6 +16315,8 @@ items:
           href: api/site-list-operations.md
         - name: Get site settings
           href: api/sitesettings-get.md
+        - name: List all root sites
+          href: api/sites-getallsites.md
```
Comment (Danielabom) [inline api-reference/beta/toc.yml:16319]:
```suggestion
          href: api/site-getallsites.md
```

File: changelog/Microsoft.FileServices.json [hunk 17]
```diff
@@ -61,6 +75,19 @@
     {
       "ChangeList": [
         {
+          "Id": "8ca9e51e-a486-41cb-af0e-105f86177d1b",
+          "ApiChange": "Method",
+          "ChangedApiName": "getAllSites",
+          "ChangeType": "Addition",
+          "Description": "Added the [getAllsites](https://learn.microsoft.com/en-us/graph/api/sites-getallsites?view=graph-rest-1.0) method to the [site](https://learn.microsoft.com/en-us/graph/api/resources/site?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.FileServices.json:82]:
```suggestion
          "Description": "Added the [getAllsites](https://learn.microsoft.com/en-us/graph/api/site-getallsites?view=graph-rest-1.0) method to the [site](https://learn.microsoft.com/en-us/graph/api/resources/site?view=graph-rest-1.0) resource.",
```

File: api-reference/beta/api/site-getallsites.md [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance.md).
```
Comment (Danielabom) [inline api-reference/beta/api/site-getallsites.md:18]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online.md).
```

File: api-reference/v1.0/api/site-getallsites.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance.md).
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-getallsites.md:16]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online.md).
```

File: api-reference/beta/api/site-list.md [hunk 20]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [sites-getallsites.md][].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[sites-getallsites.md]: sites-getallsites.md
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:29]:
```suggestion
[site-getallsites.md]: site-getallsites.md
```

File: api-reference/v1.0/api/site-list.md [hunk 21]
```diff
@@ -20,9 +20,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [sites-getallsites.md][].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[sites-getallsites.md]: sites-getallsites.md
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-list.md:27]:
```suggestion
[site-getallsites.md]: site-getallsites.md
```

File: api-reference/beta/resources/site.md [hunk 22]
```diff
@@ -27,6 +27,7 @@ The **site** resource provides metadata and relationships for a SharePoint site.
 | [Get activities by interval][]                    | GET /sites/{site-id}/getActivitiesByInterval                |
 | [List pages][]                                    | GET /sites/{site-id}/pages                                  |
 | [List root sites][]                               | GET /sites?filter=root ne null&select=siteCollection,webUrl |
+| [List root sites across geographies][]            | GET /sites/getAllSites                                      |
```
Comment (Danielabom) [inline api-reference/beta/resources/site.md:30]:
```suggestion
| [List root sites across geographies][]            | GET /site/getAllSites                                      |
```

File: api-reference/v1.0/resources/site.md [hunk 23]
```diff
@@ -36,6 +37,7 @@ The **site** resource provides metadata and relationships for a SharePoint site.
 
 [Get site]: ../api/site-get.md
 [Get root site]: ../api/site-get.md
+[List root sites across geographies]: ../api/sites-getallsites.md
```
Comment (Danielabom) [inline api-reference/v1.0/resources/site.md:40]:
```suggestion
[List root sites across geographies]: ../api/site-getallsites.md
```

File: api-reference/beta/api/site-list.md [hunk 24]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [sites-getallsites.md][].
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:25]:
```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites.md][].
```

File: api-reference/beta/resources/site.md [hunk 25]
```diff
@@ -51,6 +52,7 @@ The **site** resource provides metadata and relationships for a SharePoint site.
 [Get activities by interval]: ../api/itemactivity-getbyinterval.md
 [List pages]: ../api/sitepage-list.md
 [List root sites]: ../api/site-list.md
+[List root sites across geographies]: ../api/sites-getallsites.md
```
Comment (Danielabom) [inline api-reference/beta/resources/site.md:55]:
```suggestion
[List root sites across geographies]: ../api/site-getallsites.md
```

File: api-reference/beta/api/site-list.md [hunk 26]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites.md][].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[site-getallsites.md]: site-getallsites.md
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:29]:
```suggestion
[getallsites]: ../api/site-getallsites.md
```

File: api-reference/beta/api/site-list.md [hunk 27]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites.md][].
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:25]:
```suggestion
If you want to list all sites across all geographies, refer to [getallsites][../api/site-getallsites.md].
```

File: api-reference/beta/api/site-list.md [hunk 28]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [getallsites][../api/site-getallsites.md].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[getallsites]: ../api/site-getallsites.md
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:29]:
```suggestion
[site-getallsites]: ../api/site-getallsites.md
```

File: api-reference/beta/api/site-getallsites.md [hunk 29]
```diff
@@ -0,0 +1,94 @@
...
+## Example
+
+### Request
+
+# [HTTP](#tab/http)
+
+<!-- { "blockType": "ignored" } -->
+
+```msgraph-interactive
+GET /sites/getAllSites
```
Comment (Danielabom) [inline api-reference/beta/api/site-getallsites.md:49]:
```suggestion
GET /site/getAllSites
```

File: api-reference/beta/api/site-getallsites.md [hunk 30]
```diff
@@ -0,0 +1,94 @@
...
+|Delegated (work or school account)     | Not supported.
+|Delegated (personal Microsoft account) | Not supported.
+|Application                            | Sites.Read.All, Sites.ReadWrite.All
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /sites/getAllSites
```
Comment (Danielabom) [inline api-reference/beta/api/site-getallsites.md:37]:
```suggestion
GET /site/getAllSites
```

File: api-reference/v1.0/api/site-getallsites.md [hunk 31]
```diff
@@ -0,0 +1,92 @@
...
+|Delegated (work or school account)     | Not supported.
+|Delegated (personal Microsoft account) | Not supported.
+|Application                            | Sites.Read.All, Sites.ReadWrite.All
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /sites/getAllSites
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-getallsites.md:35]:
```suggestion
GET /site/getAllSites
```

File: api-reference/v1.0/api/site-getallsites.md [hunk 32]
```diff
@@ -0,0 +1,92 @@
...
+## Example
+
+### Request
+
+# [HTTP](#tab/http)
+
+<!-- { "blockType": "ignored" } -->
+
+```msgraph-interactive
+GET /sites/getAllSites
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-getallsites.md:47]:
```suggestion
GET /site/getAllSites
```

File: api-reference/v1.0/api/site-getallsites.md [hunk 33]
```diff
@@ -0,0 +1,92 @@
...
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online.md).
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-getallsites.md:16]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](../concepts/scan-guidance?view=odsp-graph-online.md).
```

File: api-reference/beta/api/site-getallsites.md [hunk 34]
```diff
@@ -0,0 +1,94 @@
...
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online.md).
```
Comment (Danielabom) [inline api-reference/beta/api/site-getallsites.md:18]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](../concepts/scan-guidance?view=odsp-graph-online.md).
```

File: api-reference/beta/api/site-getallsites.md [hunk 35]
```diff
@@ -0,0 +1,94 @@
...
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](../concepts/scan-guidance?view=odsp-graph-online.md).
```
Comment (Danielabom) [inline api-reference/beta/api/site-getallsites.md:18]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](../onedrive/developer/rest-api/concepts/scan-guidance.md).
```

File: api-reference/v1.0/api/site-getallsites.md [hunk 36]
```diff
@@ -0,0 +1,92 @@
...
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](../concepts/scan-guidance?view=odsp-graph-online.md).
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-getallsites.md:16]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](../onedrive/developer/rest-api/concepts/scan-guidance.md).
```

File: api-reference/beta/api/site-getallsites.md [hunk 37]
```diff
@@ -0,0 +1,94 @@
...
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](../onedrive/developer/rest-api/concepts/scan-guidance.md).
```
Comment (Danielabom) [inline api-reference/beta/api/site-getallsites.md:18]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance).
```

File: api-reference/v1.0/api/site-getallsites.md [hunk 38]
```diff
@@ -0,0 +1,92 @@
...
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# sites: getAllSites
+
+List all available [sites](../resources/site.md) across geographies in an organization.
+
+For more details, see [Best practices for discovering files and detecting changes at scale](../onedrive/developer/rest-api/concepts/scan-guidance.md).
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-getallsites.md:16]:
```suggestion
For more details, see [Best practices for discovering files and detecting changes at scale](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance).
```

File: api-reference/beta/api/site-list.md [hunk 39]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:25]:
```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites][].
```

File: api-reference/beta/api/site-list.md [hunk 40]
```diff
@@ -22,9 +22,11 @@ Specific filter criteria and query options are also supported and described belo
...
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[site-getallsites]: ../api/site-getallsites.md
 
 For more guidance about building applications that use site discovery for scanning purposes, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online&preserve-view=true).
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:31]:
```suggestion
For more guidance about building applications that use site discovery for scanning purposes, see [Best practices for discovering files and detecting changes at scale](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance).
```

File: api-reference/v1.0/api/site-list.md [hunk 41]
```diff
@@ -20,9 +20,11 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[site-getallsites]: ../api/site-getallsites.md
 
 For more guidance about building applications that use site discovery for scanning purposes, see [Best practices for discovering files and detecting changes at scale](/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online&preserve-view=true).
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-list.md:29]:
```suggestion
For more guidance about building applications that use site discovery for scanning purposes, see [Best practices for discovering files and detecting changes at scale](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance).
```

File: api-reference/beta/api/site-list.md [hunk 42]
```diff
@@ -22,11 +22,13 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[site-getallsites]: ../api/site-getallsites.md
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:29]:
```suggestion
[getallsites]: ../api/site-getallsites.md
```

File: api-reference/v1.0/api/site-list.md [hunk 43]
```diff
@@ -20,11 +20,13 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].
 
 [$search]: site-search.md
 [sites]: ../resources/site.md
+[site-getallsites]: ../api/site-getallsites.md
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-list.md:27]:
```suggestion
[getallsites]: ../api/site-getallsites.md
```

File: api-reference/beta/api/site-list.md [hunk 44]
```diff
@@ -22,11 +22,13 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].
```
Comment (Danielabom) [inline api-reference/beta/api/site-list.md:25]:
```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites][].
```

File: api-reference/v1.0/api/site-list.md [hunk 45]
```diff
@@ -20,11 +20,13 @@ Specific filter criteria and query options are also supported and described belo
 |`siteCollection/root ne null` | `siteCollection,webUrl` | Lists all root-level site collections in the organization. Useful for discovering the home site for each geography.
 
 In addition, you can use a **[$search][]** query against the `/sites` collection to find sites matching given keywords.
+If you want to list all sites across all geographies, refer to [site-getallsites][../api/site-getallsites.md].
```
Comment (Danielabom) [inline api-reference/v1.0/api/site-list.md:23]:
```suggestion
If you want to list all sites across all geographies, refer to [site-getallsites][].
```

**Comment by Danielabom** [comment]
Dear @tushar20 ,
I am reaching out to you today regarding the GitHub pull request #PR 19923 that we have been working on. We have noticed some build issues that need your attention to ensure the successful completion of the PR. As an FYi, I’ve fixed most of the issues, but need some collaboration from your side. 
These build issues are blocking the PR from being merged, and it is crucial that we resolve them as soon as possible. Your expertise and attention to detail are greatly appreciated in resolving these problems effectively.

To assist you in troubleshooting the build issues, I recommend the following steps:

1.	Review the error messages or logs provided to gain a deeper understanding of the problem.

2.	Examine the changes made in the PR and identify any potential conflicts or compatibility issues with the existing codebase.

3.	Verify that all necessary dependencies and libraries are correctly included and up to date.

4.	Attempt to reproduce the build issues locally to gain insights into their root causes.

5.	Make the necessary adjustments and modifications to address the identified build issues.

Once you have made the required fixes, please request a review and I’ll get the PR back. 

Thank you in advance for your prompt attention to these build issues.

**Pull request merged**